Playgrounds/Argo Playground.playground/Pages/Basics.xcplaygroundpage/timeline.xctimeline

```diff
@@ -3,122 +3,122 @@
    version = "3.0">
    <TimelineItems>
       <LoggerValueHistoryTimelineItem
-         documentLocation = "#CharacterRangeLen=0&amp;CharacterRangeLoc=19&amp;EndingColumnNumber=9&amp;EndingLineNumber=45&amp;StartingColumnNumber=1&amp;StartingLineNumber=45&amp;Timestamp=455213762.601599"
+         documentLocation = "#CharacterRangeLen=0&amp;CharacterRangeLoc=0&amp;EndingColumnNumber=9&amp;EndingLineNumber=67&amp;StartingColumnNumber=1&amp;StartingLineNumber=67&amp;Timestamp=455496627.584604"
```

Comment:
Should these files be ignored? I'm not sure what they are doing, but I have a suspicion that they are just noise.